xml/article_nfs_storage.xml

```diff
@@ -288,15 +288,17 @@
        </callout>
        <callout arearefs="co-ha-quick-nfs-fencing-policy">
         <para>
-         Specifies the fencing policy. For clusters with a &stonith; device
-         configured, use <literal>resource-and-stonith</literal>.
+         Specifies the fencing policy. To be supported, &sleha; clusters must have a
+         &stonith; device configured. Therefore, use <literal>resource-and-stonith</literal>.
```

Comment:
It makes sense for me to change it as "Specifies the fencing policy at the drbd level to suspend active io immediately, if any, till STONITH completes, for example. ", and drop the remaining part since it is about the cluster level fencing and it is not the scope of drbd.

Reply:
Thanks @zzhou1! How about: "Specifies the fencing policy at the DRBD level. Use `resource-and-stonith` to immediately suspend active I/O operations until STONITH completes, for example."

Reply:
I tweak it a little

"Specifies the fencing policy "resource-and-stonith" at the DRBD level. It immediately suspends active I/O operations until STONITH completes, for example." 

since resource-and-stonith might have some other subtitle use case scenarios in the code and not documented ...

Reply:
Sounds good to me, thanks!